key-concepts/covenants.adoc

```diff
@@ -1,11 +1,174 @@
 = Covenants
 
-A covenant is a set of link:actions.adoc[action] creators and types, a
-reducer (which is a link:smart_contracts.adoc[smart contract], and a
-Redux link:sagas.adoc[saga] that together indicate the functioning of an
-Interbit blockchain. These actions, smart contract, and saga work
-together to fulfill the functionality of a single Interbit blockchain.
-The `rootSaga` in a smart contract is optional.
+[quote,Merriam Webster Dictionary]
+**covenant**: a written agreement or promise, usually under seal,
+between two or more parties, especially for the performance of some
+action.
+
+Interbit blockchain networks are governed by covenants. Each node in the
+blockchain's network agrees to apply the blockchain's covenant.
```

Comment:
~Each node in the blockchain's network agrees to apply the blockchain's covenant.~ The validator nodes running the blockchain must agree to apply the covenant to the blockchain.

This sentence is ambiguous because not all nodes participate in deciding which covenant will run. Only validators (in PoA only the blockmaster) participate in the decision. Furthermore, not all validator nodes in a blockchain's network will decide which covenant runs on any particular chain because the validators in the network may actually be validators for a different blockchain.